vnext/Microsoft.ReactNative/Views/ConfigureBundlerDlg.cpp

```diff
@@ -0,0 +1,180 @@
+                                                                 .as<xaml::FrameworkElement>()
+                                                                 .FindName(L"Port")
+                                                                 .as<xaml::Controls::TextBox>()
+                                                                 .Text())));
+            options.DeveloperSettings.SourceBundleName =
+                Microsoft::Common::Unicode::Utf16ToUtf8(m_flyout.Content()
+                                                            .as<xaml::FrameworkElement>()
+                                                            .FindName(L"EntryPoint")
+                                                            .as<xaml::Controls::TextBox>()
+                                                            .Text()
+                                                            .c_str());
+            Hide();
+            reactHost->ReloadInstanceWithOptions(std::move(options));
+          }
+        });
+
+    auto resetButton = configureUI.FindName(L"Reset").as<xaml::Controls::Button>();
+    m_resetRevoker = resetButton.Click(
+        winrt::auto_revoke, [this](auto const & /*sender*/, xaml::RoutedEventArgs const & /*args*/) noexcept {
+          if (auto reactHost = m_weakReactHost.GetStrongPtr()) {
+            auto options = reactHost->Options();
+            options.DeveloperSettings.SourceBundleHost = facebook::react::DevServerHelper::DefaultPackagerHost;
+            options.DeveloperSettings.SourceBundlePort = facebook::react::DevServerHelper::DefaultPackagerPort;
+            options.DeveloperSettings.SourceBundleName = nullptr;
+            Hide();
+            reactHost->ReloadInstanceWithOptions(std::move(options));
+          }
+        });
+
+    auto cancelButton = configureUI.FindName(L"Cancel").as<xaml::Controls::Button>();
+    m_cancelRevoker = cancelButton.Click(
+        winrt::auto_revoke, [this](auto const & /*sender*/, xaml::RoutedEventArgs const & /*args*/) noexcept {
+          Hide();
+        });
+
+    m_flyout = xaml::Controls::Flyout{};
+    m_flyout.Content(configureUI);
+    if (react::uwp::Is19H1OrHigher()) {
+      // ShouldConstrainToRootBounds added in 19H1
+      m_flyout.ShouldConstrainToRootBounds(false);
+    }
+
+    configureUI.Tag(*this);
+    m_flyout.Closing(
+        [](xaml::Controls::Primitives::FlyoutBase const &sender,
+           xaml::Controls::Primitives::FlyoutBaseClosingEventArgs const & /*args*/) noexcept {
+          // Remove the tag to disconnect the ref cycle
+          sender.as<xaml::Controls::Flyout>().Content().as<xaml::FrameworkElement>().Tag(nullptr);
+        });
+
+    xaml::UIElement root{nullptr};
+    auto xamlRoot = winrt::Microsoft::ReactNative::XamlUIService::GetXamlRoot(reactHost->Options().Properties);
+    if (xamlRoot) {
+      m_flyout.XamlRoot(xamlRoot);
+      root = xamlRoot.Content();
+    } else {
+      auto window = xaml::Window::Current();
```

Comment:
this and the flyout stuff at the top is duplicated in the main devmenu too, we should keep in mind to put it in a base class when we add the 3rd flyout.